eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $1 ]; then
+    echo "usage: $0 <deployment-name> <steps>"
+    echo "eg: $0 test1"    
+    exit 1
+fi
+
+on_error() {
+    set +e
+    echo "There was an error, execution halted" >&2
+    echo "Error at line $1"
+    exit 1
+}
+
+trap 'on_error $LINENO' ERR
+
+export PREFIX=$1
+export RESOURCE_GROUP=$PREFIX
+export LOCATION=eastus
+
+# 10000 messages/sec
```

Comment:
Need to delete these from here and parent create-solution.sh as well, its causing confusion.

Reply:
I think this is helpful to leave here. Did you get feedback that it causes confusion?

Reply:
RESOURCE_GROUP  is actually used.

Reply:
I meant the duplicate "# 10000 messages/sec" section with different numbers is creating confusion. Users wont know which one to uncomment to actually repro the scenario.

Reply:
Removed the duplicate 10000 messages/sec section